packages/flutter/lib/src/widgets/default_text_editing_shortcuts.dart

```diff
@@ -431,4 +427,32 @@ class DefaultTextEditingShortcuts extends Shortcuts {
         return _windowsShortcuts;
     }
   }
+
+  @override
+  Widget build(BuildContext context) {
+    Widget result = child;
+    if (kIsWeb) {
+      // On web platform, these shortcuts makes sure the following:
+      //
+      // 1. Shortcuts fired when a TextField is focused are ignored and
+      //    forwarded to the browser.
+      // 2. Shortcuts fired when no TextField is focused will still trigger
+      //    default text editing Shortcuts
```

Comment:
Shortcuts fired when no EditableText is focused will still trigger _shortcuts assuming DoNothingAndStopPropagationTextIntent is unhandled elsewhere.